java/src/org/openqa/selenium/bidi/emulation/SetGeolocationOverrideParameters.java

```diff
@@ -0,0 +1,77 @@
+// Licensed to the Software Freedom Conservancy (SFC) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The SFC licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package org.openqa.selenium.bidi.emulation;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class SetGeolocationOverrideParameters {
+  private final GeolocationCoordinates coordinates;
+  private final GeolocationPositionError error;
+  private final List<String> contexts;
+  private final List<String> userContexts;
+
+  public SetGeolocationOverrideParameters(
+      GeolocationCoordinates coordinates,
```

Comment:
```
emulation.SetGeolocationOverrideParameters = {
  (
    (coordinates: [emulation.GeolocationCoordinates](https://www.w3.org/TR/webdriver-bidi/#cddl-type-emulationgeolocationcoordinates) / [null](https://www.rfc-editor.org/rfc/rfc8610#appendix-D)) //
    (error: [emulation.GeolocationPositionError](https://www.w3.org/TR/webdriver-bidi/#cddl-type-emulationgeolocationpositionerror))
  ),
  ? contexts: [+[browsingContext.BrowsingContext](https://www.w3.org/TR/webdriver-bidi/#cddl-type-browsingcontextbrowsingcontext)],
  ? userContexts: [+[browser.UserContext](https://www.w3.org/TR/webdriver-bidi/#cddl-type-browserusercontext)],
}  
```
I might be wrong, but it seems like either coordinates or error can be set. Is that the case? or both can be set?